common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1332,6 +1333,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("3s")
 
+  val EXCLUDED_WORKER_EXPIRED_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.excludedWorker.expired.timeout")
```

Comment:
we already have a `celeborn.worker.excluded.checkInterval` ...

Reply:
> we already have a `celeborn.worker.excluded.checkInterval` ...

Different usage

Reply:
I mean the naming style

Reply:
> I mean the naming style

Got it, updated